packages/faceted-search/CHANGELOG.md

```diff
@@ -26,6 +26,15 @@ Types of changes
 
 ## [unreleased]
 
+### Breaking change
+Rename two data-feature attributes for harmonization:
+- `faceted-advanced-search.cancel` become `filter.advanced.clear`
+- `faceted-advanced-search.submit` become `filter.advanced.apply`
+
+### Added
+
+- [feat](https://github.com/Talend/ui/pull/2759): Add somes data-feature attributes for Pendo tracking / E2E tests
```

Comment:
you can just write `#2759` instead of the full url and github does the link

Reply:
Just test, `[feat](#2759) not work: it take this for an anchor in Readme.md.
Think it can be work if i just do a `#2759` but we want to keep a label like "feat", "fix" etc..

Anyway, #2759 will only works in a Github context, can be good to keep a full link to copy-paste from IDE.